### PR TITLE
doc: added tz section to access log

### LINKED ISSUE #7177: Add section covering Time Zones for AccessLog in v2
### Do you want to request a *feature* or report a *bug*?

Feature

### What did you expect to see?

https://github.com/containous/traefik/blob/v1.7/docs/configuration/logs.md#time-zones in 2.x documentation.

In the 1.7 documentation, there is a brief section covering how to enable the local server timezone for access logs that appears to have never been ported to 2.x. The functionality seems to still exist and was confirmed as functioning per @kevinpollet and was re-discovered per https://github.com/containous/traefik/issues/7163

### PR DESCRIPTION
**Pull request by notsureifkevin**
### What does this PR do?

Fixes #7177 

### Motivation

NO-yeast-Homemade-donuts-FINAL

### More

- [ ] Added/updated tests
- [X] Added/updated documentation

### CODE REVIEW

File: docs/content/observability/access-logs.md [hunk 1]
```diff
@@ -217,3 +217,34 @@ This allows the logs to be rotated and processed by an external program, such as
...
+
+## Time Zones
+
+Traefik will timestamp each log line in UTC time by default.
+
+It is possible to configure the Traefik to timestamp in a specific timezone by ensuring the following configuration has been made in your environment:
+
+1. Provide time zone data to `/etc/localtime` or `/usr/share/zoneinfo` (based on your distribution)
+2. Set the environment variable TZ to the desired timezone
+3. Specify the field `StartLocal` by dropping the field named `StartUTC` (available on the default Common Log Format (CLF) as well as JSON)
```
Comment (rtribotte) [inline docs/content/observability/access-logs.md:229]: It seems that it's only needed to set either the `/etc/localtime` or the environment variable TZ (and the TZ env var overrides the `/etc/localtime`)

```suggestion
1. Provide time zone data to `/etc/localtime` or `/usr/share/zoneinfo` (based on your distribution) or set the environment variable TZ to the desired timezone
2. Specify the field `StartLocal` by dropping the field named `StartUTC` (available on the default Common Log Format (CLF) as well as JSON)
```

File: docs/content/observability/access-logs.md [hunk 2]
```diff
@@ -217,3 +217,34 @@ This allows the logs to be rotated and processed by an external program, such as
...
+    environment:
+      - TZ=US/Alaska
+    command:
+      - --accesslog
+      - --accesslog.fields.names.StartUTC=drop
+      - --providers.docker
+    ports:
+      - 80:80
+    volumes:
+      - /etc/localtime:/etc/localtime
```
Comment (rtribotte) [inline docs/content/observability/access-logs.md:248]: Since it is not mandatory to have both, maybe we can remove:
```suggestion
```

**Review by rtribotte — APPROVED** [review]
LGTM 👍

**Review by ldez — APPROVED** [review]
LGTM

**Review by juliens — APPROVED** [review]
LGTM

**Pull request merged**